src/main/java/mil/dds/anet/database/PositionDao.java

```diff
@@ -609,24 +610,39 @@ public int mergePositions(Position winner, Position loser) {
     return nr;
   }
 
-  private void updatePeoplePositions(final String positionUuid, final String personUuid,
-      final Instant startTime, final Instant endTime) {
-    if (endTime == null) {
-      // we have to make an exception here, as MSSQL has problems inserting a null datetime
-      getDbHandle()
-          .createUpdate("INSERT INTO \"peoplePositions\" "
-              + "(\"positionUuid\", \"personUuid\", \"createdAt\") "
-              + "VALUES (:positionUuid, :personUuid, :createdAt)")
-          .bind("positionUuid", positionUuid).bind("personUuid", personUuid)
-          .bind("createdAt", DaoUtils.asLocalDateTime(startTime)).execute();
-    } else {
-      getDbHandle()
-          .createUpdate("INSERT INTO \"peoplePositions\" "
-              + "(\"positionUuid\", \"personUuid\", \"createdAt\", \"endedAt\") "
-              + "VALUES (:positionUuid, :personUuid, :createdAt, :endedAt)")
-          .bind("positionUuid", positionUuid).bind("personUuid", personUuid)
-          .bind("createdAt", DaoUtils.asLocalDateTime(startTime))
-          .bind("endedAt", DaoUtils.asLocalDateTime(endTime)).execute();
+  @InTransaction
+  public int updatePositionPreviousPeople(Position pos) {
+    final String posUuid = pos.getUuid();
+    // Delete old history
+    final int numRows = getDbHandle()
+        .execute("DELETE FROM \"peoplePositions\"  WHERE \"positionUuid\" = ?", posUuid);
+    // Add new history
+    for (final PersonPositionHistory history : pos.getPreviousPeople()) {
+      AnetObjectEngine.getInstance().getPersonDao().updatePeoplePositions(posUuid,
+          history.getPersonUuid(), history.getStartTime(), history.getEndTime());
+    }
+    return numRows;
+  }
+
+  public boolean hasPersonConflict(Position pos) {
```

Comment:
Code looks very similar to `PersonDao::hasPositionConflict`; should probably use that method.

Reply:
it is similar but not the same.